### PR TITLE
Update ggplot URLs

### PR DESCRIPTION
**Pull request by andreas-h**
see #6859. while at it, changed link to R to https

### CODE REVIEW

File: examples/style_sheets/plot_ggplot.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 .. [1] http://www.huyng.com/posts/sane-color-scheme-for-matplotlib/
 
-.. _ggplot: http://had.co.nz/ggplot/
-.. _R: http://www.r-project.org/
+.. _ggplot: https://github.com/yhat/ggplot
```
Comment (tacaswell) [inline examples/style_sheets/plot_ggplot.py]: This is the python library built on top of mpl, not the R library.

Comment (jenshnielsen) [inline examples/style_sheets/plot_ggplot.py]: I think this is the right url http://ggplot.yhathq.com/

Comment (andreas-h) [inline examples/style_sheets/plot_ggplot.py]: sorry, my bad. fixed to http://ggplot2.org/  I'm not sure about the `2`, though. But it looks right.

**Comment by tacaswell** [comment]
backported to v1.5.2-doc as b6192a6

**Comment by tacaswell** [comment]
:tada: 

From github stalking it looks like this is your first contribution in ~4yrs.  

Thank you and hopefully we will hear from you again in < 4 years.

**Pull request merged**